Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -378,6 +366,22 @@ open class NavigationMapView: UIView {
         navigationCamera.follow()
     }
     
+    func updateUserCourseViewWithAccuracy() {
+        let frame = CGRect(origin: .zero, size: 75.0)
+        let isHidden = userCourseView.isHidden
+        switch userLocationStyle {
+        case .courseView(let courseView):
+            userCourseView = reducedAccuracyActivatedMode
+                ? UserHaloCourseView(frame: frame)
+                : courseView
+        default:
```

Comment:
Explicitly match the `puck2D` and `puck3D` cases here, so that the compiler reminds us to double-check this code if we ever add another case to the enumeration. There are a couple of similar default cases on `UserLocationStyle` elsewhere in this file.

Reply:
Thanks for the suggestion, I'll add the explicit cases to avoid future error.